app.js

```diff
@@ -65,6 +65,7 @@ async function setAuthors(apiURL) {
     personDiv.innerHTML = `
                 <div class="name_and_img">
                     <img src="${author.avatar}" alt="${author.name}'s avatar">
+                     <a href="${contributor.html_url}" class="tooltip" target="_blank">
```

Comment:
Hi! Thank you for your contribution 🙌 

Can you please share a screenshot or a short screen recording showing how it looks?

Also, did you try making the whole `name_and_img` a link? Let's try letting the user click on the avatar as well.